frontend/src/components/common/checkbox/checkbox.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import styles from './styles.module.scss';
+
+interface CheckboxType {
+  options: Array<string>;
+  onChange(value: string): void;
+}
```

Comment:
Common checkbox should be only one (without any mappings)

```suggestion
interface Props {
  onChange: InputChangeEvent;
  isChecked: boolean;
}
```